README.md

```diff
@@ -48,7 +48,7 @@ The bastion host includes an OpenVPN instance easily manageable by using [furyag
 ### Requirements
 
 - **AWS Access Credentials** of an AWS Account with the following [IAM permissions](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/docs/iam-permissions.md).
-- **terraform** `0.15.4`
+- **terraform** `>=1.3.0`
```

Comment:
what's the reasoning for choosing `1.3.0` and not `1.4.6` (the latest as of today)?

Reply:
The `1.3.0` is the first terraform version to support the `optional` attribute for `object` in GA.
Using this version allows us to remove the `experiments` option from terraform block and support future Terraform versions.

Reply:
1.4.6 also allows us to remove the experiments option, doesn't it?

Reply:
Yes, it does

Reply:
Terraform 1.4 has native support for `null_resource` that I think we use, it could be worthy to consider setting 1.4 as minimum: https://www.hashicorp.com/blog/terraform-1-4-improves-the-cli-experience-for-terraform-cloud

Reply:
Terraform's native support for `null_resource` means that it will be replaced by `terraform_data`. Currently we do not use it anywhere. If we use it in a module in the future, the `required_version: ">=1.4.0"` will force the use of the newer version.

Reply:
We do use the `null` provider and `null_resource`, see:
https://github.com/sighupio/fury-eks-installer/blob/c9b845dd4c5d13205af0f0c9d5f92c1a7cc6df8b/modules/vpc-and-vpn/vpn.tf#L179
https://github.com/sighupio/fury-eks-installer/blob/c9b845dd4c5d13205af0f0c9d5f92c1a7cc6df8b/modules/vpc-and-vpn/vpn.tf#L188

Reply:
> Terraform 1.4 introduces a new `terraform_data` resource as a built-in replacement for the null resource. This removes the need to include the null provider in your configuration and supports all the same capabilities as `null_resource`.

A change to the code is required to use the built-in native resource, and represents a "breaking change" that must be handled with a `terraform state mv`